src/DocumentFormat.jl

```diff
@@ -18,7 +20,11 @@ mutable struct FormatOptions
     lineends::Bool
     kw::Bool
 end
-FormatOptions() = FormatOptions(4, true, true, true, true, true, true, true, true, true, true)
+FormatOptions() = FormatOptions(default_options...)
+
+# Default to 4-space indentation with all options on
```

Comment:
Crap. I forgot to delete this clearly inaccurate comment. :/

Reply:
No  worries, follow up PR is fine :) We merge fast here ;)